src/jdk.jdeps/share/classes/com/sun/tools/jdeps/DependencyFinder.java

```diff
@@ -268,7 +268,14 @@ boolean isParsed(Location location) {
             }
             return targets;
         } catch (InterruptedException|ExecutionException e) {
-            throw new Error(e);
+            Throwable t = e.getCause();
+            if (t instanceof RuntimeException) {
+                throw (RuntimeException)t;
+            } else if (t instanceof Error) {
+                throw (Error)t;
+            } else {
+                throw new Error(e);
```

Comment:
A minor suggestion is that you could avoid the casts with:

Throwable cause = ...
if (cause instanceof RuntimeException e) {
    throw e;
} else if (cause instanceof Error e) {
    throw e;
} else {

Reply:
Fixed.  Thanks for reminding.